Sources/ArcGISToolkit/Components/Authentication/CertificatePickerViewModifier.swift

```diff
@@ -181,14 +184,22 @@ struct CertificatePickerViewModifier: ViewModifier {
                 continueAction: .init(
                     title: String(localized: "OK", bundle: .toolkitModule),
                     handler: { _, password in
-                        viewModel.proceed(withPassword: password)
+                        viewModel.proceedToUseCertificate(withPassword: password)
                     }
                 )
             )
-            .alertCertificateError(
+            .certificateErrorSheet(
                 isPresented: $viewModel.showCertificateError,
                 viewModel: viewModel
             )
+            .task {
+                // Present the prompt right away.
+                // Setting it after initialization allows it to animate.
+                // However, this needs to happen after a slight delay or
+                // it doesn't show.
+                try? await Task.sleep(nanoseconds: 250_000_000)
+                viewModel.showPrompt = true
+            }
```

Comment:
I see this task is used 3x throughout. Would a custom modifier potentially be better? The doc comment will loose the context but that could potentially be alleviated by good documentation of the new modifier: e.g.

```swift
/// Performs the provided action after a slight delay.
/// - Tip: Occasionally delaying allows a sheet's presentation animation to work correctly.
/// - Parameters:
///   - delay: The delay to wait before allow the task to proceed, in nanoseconds. Defaults to 1/4 second.
///   - action: The action to perform after the delay.
func delayedTask(delay: UInt64 = 250_000_000, action: @escaping () -> Void) -> some View {
    task {
        try? await Task.sleep(nanoseconds: delay)
        action()
    }
}
```

Reply:
I think that's a great idea to explore. Let me look into that.

Reply:
I think the title of the method needs to be delayedOnAppear because the action is not async so it's not really the same as task.  I incorporated your idea with some slight modifications.